pages/get_involved.md

```diff
@@ -7,57 +7,105 @@ subtitle: Ways to contribute to the US-RSE Association
 
 The US-RSE Association is a new organization that needs community involvement to grow and mature.  Below are some ways members can get more involved in building this organization. Have additional ideas for a working group or committee?  Let us know and bring the discussion to slack.
 
-If you have another idea but aren’t sure how to get started, the Steering Committee would be happy to help. Reach out on Slack or email us all at [contact@us-rse.org](mailto:contact@us-rse.org).
+If you have another idea but aren’t sure how to get started, the Steering Committee would be happy to help. Reach out on Slack (@sc) or email us all at [contact@us-rse.org](mailto:contact@us-rse.org).
+
+<br>
+
+# Easy
 
 ---
 
-## US-RSE Website Committee Charter
+### Invite a New Member
 
-We are seeking help to develop a charter document and determine the initial members of the US-RSE Website Committee. Once ready, the document will be reviewed and ratified by the US-RSE Steering Committee.
+Email at least one other person who might be interested in the US-RSE association and invite them to join (via the link on the main website page: [https://us-rse.org/](https://us-rse.org/)).  
 
-There are two main goals for the committee: 1. Provide guidance for website growth and direction 2. Technical website management
+### Introduce Yourself
 
-As the US-RSE association grows, it is important that we have the proper guidance and support structure in place to ensure the website reflects the values and beliefs of the organization as a whole.
+Have you been lurking on Slack for a while?  Introduce yourself on the #introductions channel, even if you’re not new to the group.
 
-* **Get involved**: join the [#website channel on Slack](https://usrse.slack.com/messages/CJTL0MWUE) 
-* **Contact**: Lance Parsons <[lparsons@princeton.edu](mailto:lparsons@princeton.edu)>.
+### Welcome New Members
 
----
+Join the #introductions channel on Slack and welcome new members.  
+
+### Share a Resource/Ask a Question
+
+Found a good resource that’s worth sharing? Post it on Slack! Looking for a good resource or some advice?  Post it on Slack!  Looking for a little more guidance? See How to Share with the Community
 
-## Membership and Voting
 
-The US-RSE aims to hold an election for members of the Steering Committee by June 2020.  In addition, we need a mechanism for occasional membership votes on issues like organizational rules. To make this happen, we need to:
-1. Define what makes people a member
-2. Have some process for tracking members
-3. Have a mechanism to allow voting
-4. Develop rules around the quorum for votes and voting rules
+<br>
 
-* **Get involved**: join the [#membership-voting channel on Slack](https://usrse.slack.com/messages/CP2LK32AZ)
-* **Contact**: Christina Maimone <[christina.maimone@northwestern.edu](mailto:christina.maimone@northwestern.edu)>
+
+# One-Time Commitment
```

Comment:
Just so the headers are clear in what they are describing, I changed the first to Quick contribution, and then it sort of followed to have the others be in similar format (One-time contribution, and Ongoing or Extended contribution)
```suggestion
# One-Time Contribution
```